src/current/v24.1/authentication.md

```diff
@@ -303,6 +301,44 @@ As discussed [earlier](#certificate-authority), the CA's public key is widely di
 
 Let's see how the digital certificate is used in client-server communication: The client (e.g., a web browser) has the CA certificate (containing the CA's public key). When the client receives a server's certificate signed by the same CA, it can use the CA certificate to verify the server's certificate, thus validating the server's identity, and securely connect to the server. The important thing here is that the client needs to have the CA certificate. If you use your own organizational CA instead of a publicly established CA, you need to make sure you distribute the CA certificate to all the clients.
 
+## Supported cipher suites
```

Comment:
These lists of supported cipher suites are all the same, they could be moved into a common include file similar to what you did with `tls-bad-cipher-warning.md`

I had to load them all into `diff` to find that out, though - they're long enough it's not really feasible to visually inspect them for differences

The only difference I could find is that the language in the v22.2 file changes for v23.1 - otherwise all identical

```
24c24
< The following cipher suites are rejected by default because they are not recommended by the IETF ([RFC 8447](https://datatracker.ietf.org/doc/html/rfc8447)). If you cannot use one of the default cipher suites, but you can use one of the disabled ones, you can set the environment variable `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` to `true`. You should set this environment variable only if you cannot use one of the default cipher suites, but you can use one of the disabled ones.
---
> The following cipher suites are rejected by default because they are not recommended by the IETF ([RFC 8447](https://datatracker.ietf.org/doc/html/rfc8447)). To enable them, set the environment variable `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` to `true`. You should set this environment variable only if you cannot use one of the default cipher suites, but you can use one of the disabled ones.
```

this could be solved by making this a common include file with a bit of version-specific logic in it.  What do you think?  the liquid logic would also serve as good documentation of "what versions have what cipher suites" whereas right now there's no way to know except manually diffing them